### PR TITLE
Email verification controller

### PR DESCRIPTION
**Pull request by EzimetYusup**
- updated config to have Path to email verification Controller component.
 - Moved Authenticate middleware to the EmailVerificationController

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -26,10 +26,10 @@
             ->name("verification.notice");
 
         Route::get("email/verify/{id}/{hash}", [
-            EmailVerificationController::class,
+            config('filament-breezy.email_verification_controller_path'),
```
Comment (Saifallak) [inline routes/web.php:29]: isn't this a breaking change to use this without default value?
as other (already running) apps that don't have this config will receive null?
or am i wrong?

Comment (Saifallak) [reply routes/web.php:29]: TBH, I didn't test it, just asking, how laravel handling this? does it merge package config with user config?

Comment (jeffgreco13) [reply routes/web.php:29]: It is. I am reviewing now. Thank you!

Comment (Saifallak) [reply routes/web.php:29]: anytime ^^, thanks for your hard work.

**Pull request merged**